src/footer/footer.macros.html

```diff
@@ -7,53 +7,42 @@
 {#
   * Renders a standard or simplified footer
 #}
-{% macro footer(simple=false, class=null, id=null, phone_number='888–498–0003') %}
+{% macro footer(data, simple=false, class=null, id=null) %}
 <footer class="footer{% if simple %} footer--simple{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
   <div class="footer-top r-inner">
     {{ caller() }}
   </div>
   <div class="footer-bottom">
-    <nav class="footer-nav r-inner">
-      <div class="footer-nav-group footer-nav-group--bottom">
-        <span class="footer-nav-country-icons">
-          <a class="footer-nav-group-link footer-nav-group-link--country" href="?country_id=49">
-            {{ icon('usa', size='small') }}
+    <div class="footer-bottom-inner r-inner">
+
+      <span class="footer-bottom-item site-copyright">
+        &copy;
+        <script>document.write(new Date().getFullYear());</script>
+      </span>
+
+      <span class="site-legal-links footer-bottom-item">
+        <a class="" href="https://casper.com/privacy">{{ data.privacy }}</a>
+        <a class="" href="https://casper.com/terms">{{ data.terms }}</a>
+      </span>
+
+      <span class="site-address h-card footer-bottom-item">
+        <span class="p-street-address">{{ data.address }}</span>
+
+        {% if not simple %}
+        <span class="site-address-meta">
+          <a class="footer-bottom-item" href="tel:{{ data.phone_number|replace('-', '') }}">
```

Comment:
Not so hot on `footer-bottom-item` either, but I need a class to target each `item` for spacing. This may change once design looks at this...less urgent now.